cardano-testnet/src/Testnet/Defaults.hs

```diff
@@ -181,7 +182,7 @@ defaultConwayGenesis =
                     , ucppGovActionDeposit = Coin 1_000_000
                     , ucppDRepDeposit = Coin 1_000_000
                     , ucppDRepActivity = EpochInterval 100
-                    , ucppMinFeeRefScriptCostPerByte = undefined
+                    , ucppMinFeeRefScriptCostPerByte = 0 %! 1 -- FIXME GARBAGE VALUE
```

Comment:
I'm unsure about this `0` here. This needs to be updated after benchmarking. Same applies to genesis files below.